src/assets/scss/CategoryCard.scss

```diff
@@ -6,12 +6,14 @@
   border: 0.1rem solid #eee;
   padding: 1.5rem;
   border-radius: 0.4rem;
+  transition: all 0.3s ease-in-out;
   display: flex;
-  transition: transform 0.1s ease-in-out;
+  box-shadow: 0 0 2.5rem 0.5rem rgba(204,204,204,0.5);
 }
 
 .category:hover {
-    box-shadow: 0px 0px 10px 2px rgba(204,204,204,0.5);
+    box-shadow: 0 1.25rem 1.25rem -0.9375rem rgba(0, 0, 0, 0.3);
+    transform: translateY(-5px);
```

Comment:
@puritybirir is there a way we can make this values `0 0 2.5rem 0.5rem rgba(204,204,204,0.5)` and  `0 1.25rem 1.25rem -0.9375rem rgba(0, 0, 0, 0.3)` variables ?
Seeing we are using it in different places.

Reply:
Done